opendatahub-io/odh-dashboard
# [RHOAIENG-2295] Convert API - Create/duplicate runs & recurring runs

Closes: [RHOAIENG-2295](https://issues.redhat.com/browse/RHOAIENG-2295)

## Description
Added ability to:
- Create runs with input parameters
- Create recurring runs & with input parameters
- Duplicate runs and recurring runs

## How Has This Been Tested?
manual + cypress tests

## How to test
Try creating both runs and recurring runs with and without input parameters. Verify you can upload new pipelines and versions within the create and duplicate forms, and that when duplicating, fields are pre-filled (input params, pipeline, version). Also, when selecting a new pipeline, the latest pipeline version is selected, and consequently all version related input params are loaded (will be blank for create form and should pre-fill for duplicate form).

Self checklist (all need to be checked):

If you have UI changes: 

After the PR is posted & before it merges:
